elb_with_ssl_no_s3logs/variables.tf

```diff
@@ -4,12 +4,12 @@ variable "environment" {}
 
 variable "name" {}
 
-variable "backend_sg" {
+variable "backend_security_groups" {
+  description = "The security groups of the ELB backends instances"
   type = "list"
 }
 
 # https://github.com/hashicorp/terraform/issues/10857
-variable "backend_sg_count" {}
```

Comment:
why remove this? afaik the bug is not fixed and you will have the same problem with your current implementation of getting the length of a list to set the count.

Reply:
I thought so too, but according @venturel, that case is fixed and working.

Reply:
We are using this already in other modules